clang/test/Driver/cuda-windows-cuda-path.cpp

```diff
@@ -0,0 +1,11 @@
+///
+/// Make sure that CUDA_PATH is picked up correctly when looking for CUDA
+/// instalation.
+///
+
+// REQUIRES: system-windows
+// RUN: env CUDA_PATH=%S\Inputs\CUDA_111\usr\local\cuda %clang -fsycl \
+// RUN: -fsycl-targets=nvptx64-nvidia-cuda -### -v %s 2>&1 | \
```

Comment:
`-v` is not needed (`-###` wins)

Reply:
I've deliberately used the `-v` to get the `Found CUDA Installation` output, seems like a perfect match for this test. Don't mind dropping it and matching on for example `-internal-isystem`'s path instead?

Reply:
Ah, I see.  Thanks for the clarification.  I'm OK with leaving as-is.